src/main/java/tk/shanebee/hg/tasks/StartingTask.java

```diff
@@ -36,6 +37,11 @@ public void run() {
         timer = (timer - 5);
 
         if (timer <= 0) {
+            //clear inventory on game start
+            for (int i = 0; i < game.getGamePlayerData().getPlayers().size(); i++){
+                Player player = Bukkit.getPlayer(game.getGamePlayerData().getPlayers().get(i));
+                player.getInventory().clear();
+            }
```

Comment:
You could probably just use a foreach here instead
```suggestion
            //clear inventory on game start
            game.getGamePlayerData().getPlayers().forEach(uuid -> {
                Player player = Bukkit.getPlayer(uuid);
                assert player != null;
                player.getInventory().clear();
            });
```